packaging/installer/netdata-uninstaller.sh

```diff
@@ -24,418 +24,418 @@ ENVIRONMENT_FILE="/etc/netdata/.environment"
 INTERACTIVITY="-i"
 YES=0
 while :; do
-	case "$1" in
-	-h | --help)
-		echo "$usage" >&2
-		exit 1
-		;;
-	-f | --force)
-		INTERACTIVITY="-f"
-		shift
-		;;
-	-y | --yes)
-		YES=1
-		shift
-		;;
-	-e | --env)
-		ENVIRONMENT_FILE="$2"
-		shift 2
-		;;
-	-*)
-		echo "$usage" >&2
-		exit 1
-		;;
-	*) break ;;
-	esac
+  case "$1" in
+    -h | --help)
+      echo "$usage" >&2
+      exit 1
```

Comment:
Why do we return an error(exit 1) when somebody asks for help?

Reply:
I don't know but this was just a formatting change.

Hit the cog wheel in the Github Review UI and tick the box that says "Ignore whitespace".

Reply:
All right, thanks!